tutorial/chapter1.md

```diff
@@ -204,41 +204,29 @@ In contrast with the previous example, in the next output the listeners of `ToGr
 [2023-09-04 14:09:13.473] [info] Total frames listened: 20.
 ```
 
-## Image multiplexer example
+## Image multiplexer example 1
 
-The [datamux](../examples/chapter1/datamux/src/main.cpp) shows the replacement of the [`DataMultiplexerMiddlewareProvider`](https://github.com/klepsydra-technologies/kpsr-core/blob/main/high_performance/modules/high_performance/include/klepsydra/high_performance/data_multiplexer_middleware_provider.h) high performance API (from kpsr-core) for processing OpenCV `Mat` images using `ImageDataFactory` and `ImageMultiplexerVisionMiddlewareProvider` classes. In this sample example, we process images by resizing them to desired size and publish them. After that, the mean of these resized images is calculated and published too.
+The [datamux](../examples/chapter1/datamux/src/main.cpp) shows the replacement of the [`DataMultiplexerMiddlewareProvider`](https://github.com/klepsydra-technologies/kpsr-core/blob/main/high_performance/modules/high_performance/include/klepsydra/high_performance/data_multiplexer_middleware_provider.h) high performance API (from kpsr-core) for processing OpenCV `Mat` images using `ImageDataFactory` and `ImageMultiplexerVisionMiddlewareProvider` classes. In this sample example, we process images by resizing them to desired size and publish them. After that, the mean of these resized images is calculated and published too. In this example we are using a single subscriber with 5 listeners.
```

Comment:
In this example we are using a single subscriber with 5 listeners
->
In this example we are using a single subscriber with 5 listeners. As mentioned in [kpsr-tutorial](https://github.com/klepsydra-technologies/kpsr-tutorial/blob/main/tutorials/chapter1.md#example-5-the-data-multiplexer), all tasks will be run sequentially, which makes it ideal when mutual exclusivity is needed.